perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -1959,14 +2008,16 @@ public void setAttribute(PerunSession sess, Resource resource, Member member, At
 			return;
 		}
 		if (!workWithUserAttributes) {
+			checkAttributeSyntax(sess, member, resource, attribute);
```

Comment:
This method is unused, please remove it. We have standard method to set member-resource attribute and there is no need to support other attribute types like we do for setAttributes(). Otherwise we would have to fill method in API and other layers of AttributesManager, but there is no demand for it.

Reply:
Ok, the method should be deleted now.